code-review/README.md

```diff
@@ -31,7 +31,8 @@ Having Your Code Reviewed
 
 * Be grateful for the reviewer's suggestions. ("Good call. I'll make that
   change.")
-* Don't take it personally. The review is of the code, not you.
+* Be aware of [how hard it is to convey emotion online] and how easy it is to
+  misinterpret feedback.
```

Comment:
How about "Seek clarification of personal comments. The intended meaning may not have survived being typed."?  That suggests a positive action, rather than a change in attitude or an admonition against a thing, without leaving room for blame.